### PR TITLE
What’s new for 5.0

### PR DESCRIPTION
**Pull request by bhariharan**
@rwhitley and I reviewed it offline.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -36,14 +36,34 @@ Introduction
...
+- Hybrid apps built on Mobile SDK will now use WKWebView instead of UIWebView.
+- Extensions are now fully supported.
+- Salesforce servers are now fully ATS compliant. As a result, we have removed ATS exceptions from Mobile SDK apps.
+
 **SmartStore Enhancements**
-- To enhance performance in certain edge cases, SmartStore adds an option for serializing unusually large soup elements in external storage.
+- We have added new APIs that allow hybrid developers to create their own named databases.
+
+**SmartSync Enhancements**
+- We now allow sync down/refresh of data stored in local soups by specifying the fields to sync.
```
Comment (wmathurin) [inline readme.md]: Not sure about "local". See the same comment on the android pull request.

File: readme.md [hunk 2]
```diff
@@ -36,14 +36,34 @@ Introduction
...
+
+**Hybrid Networking Enhancements**
+- We have replaced forcetk.mobilesdk.js with force.js. Networking in hybrid apps is now handled natively. As a result, session token refresh happens automatically.
```
Comment (wmathurin) [inline readme.md]: Add "through a new plugin (com.salesforce.plugin.network)"

Comment (wmathurin) [inline readme.md]: Also we should mention smartsync.js no longer depends on jQuery but instead uses browser promises.

**Comment by bhariharan** [comment]
Done [here](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1860/commits/3a7e56dcc9324051727edf02e681657b336a5941).

**Pull request merged**